src/world/area_mac/mac_00/7EB340.c

```diff
@@ -375,7 +375,17 @@ INCLUDE_ASM(s32, "world/area_mac/mac_00/7EB340", func_80241FF0_7ECD00);
 
 #include "world/common/UnkPositionFunc.inc.c"
 
-INCLUDE_ASM(s32, "world/area_mac/mac_00/7EB340", func_80242178_7ECE88);
+// INCLUDE_ASM(s32, "world/area_mac/mac_00/7EB340", func_80242178_7ECE88);
+
+extern s32 mac_00_dojoBattleIDs[];
+extern EncounterStatus gCurrentEncounter;
```

Comment:
I don't think this extern is required; gCurrentEncounter is declared by common.h/variables.h